pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -32,7 +33,8 @@ def configure(self,
                   initial_sweeps: typing.List[AbstractSweep],
                   sweeps: typing.List[AbstractSweep],
                   sim_params: typing.Dict,
-                  file_params: typing.Dict):
+                  file_params: typing.Dict,
+                  ih_file_params: typing.Dict = None):
```

Comment:
This needs to be documented much more clearly - I don't understand why this is called 'ih_' and I think you need a sentence explaining how this is different to 'file_params' to explain to a user who doesn't know about this project - i.e. that this provides individual level outputs in a form compatible with the separate epiOS tool

Reply:
Will document this more clearly, and also will explain the variable name in the documentation, unless you want me to change the name completely? It stands for 'infection history'

Reply:
Hmm given it is very accessible I think that might be worthwhile. How about `inf_history_params`?

Reply:
Sure, will do!